common/src/main/java/com/abrenoch/hyperiongrabber/common/HyperionScreenService.java

```diff
@@ -186,6 +194,12 @@ public void onDestroy() {
         super.onDestroy();
     }
 
+    private void haltStartup() {
+        // this line feels unnecessary, but problems happen without it
```

Comment:
This is because the service will be killed otherwise, right? That's a valid reason to put it here